diff --git a/hw/xfree86/common/xf86Init.c b/hw/xfree86/common/xf86Init.c
index 017dcb6..31bbf17 100644
--- a/hw/xfree86/common/xf86Init.c
+++ b/hw/xfree86/common/xf86Init.c
@@ -1177,7 +1177,7 @@ ddxProcessArgument(int argc, char **argv, int i)
     }
 
     /* First the options that are not allowed with elevated privileges */
-    if (!strcmp(argv[i], "-modulepath") || !strcmp(argv[i], "-logfile")) {
+    if (!strcmp(argv[i], "-modulepath")) {
         if (xf86PrivsElevated()) {
             FatalError("The '%s' option cannot be used with "
                        "elevated privileges.\n", argv[i]);
