--- a/src/client/linux/dump_writer_common/thread_info.h	2016-06-17 22:29:59.869046131 +0000
+++ b/src/client/linux/dump_writer_common/thread_info.h	2016-06-17 22:33:55.665586619 +0000
@@ -66,8 +66,8 @@
   struct user_fpregs fpregs;
 #elif defined(__aarch64__)
   // Use the structures defined in <sys/user.h>
-  struct user_regs_struct regs;
-  struct user_fpsimd_struct fpregs;
+  struct user_pt_regs regs;
+  struct user_fpsimd_state fpregs;
 #elif defined(__mips__)
   // Use the structure defined in <sys/ucontext.h>.
   mcontext_t mcontext;
--- a/src/tools/linux/md2core/minidump-2-core.cc	2016-06-17 22:52:08.347018545 +0000
+++ b/src/tools/linux/md2core/minidump-2-core.cc	2016-06-17 22:58:19.398213618 +0000
@@ -157,7 +157,7 @@
   elf_timeval    pr_stime;      /* System time                               */
   elf_timeval    pr_cutime;     /* Cumulative user time                      */
   elf_timeval    pr_cstime;     /* Cumulative system time                    */
-  user_regs_struct pr_reg;      /* CPU registers                             */
+  user_pt_regs pr_reg;      /* CPU registers                             */
   uint32_t       pr_fpvalid;    /* True if math co-processor being used      */
 } prstatus;
 
@@ -216,7 +216,7 @@
 #if defined(__mips__)
     mcontext_t mcontext;
 #else
-    user_regs_struct regs;
+    user_pt_regs regs;
 #endif
 #if defined(__i386__) || defined(__x86_64__)
     user_fpregs_struct fpregs;
@@ -225,7 +225,7 @@
     user_fpxregs_struct fpxregs;
 #endif
 #if defined(__aarch64__)
-    user_fpsimd_struct fpregs;
+    user_fpsimd_state fpregs;
 #endif
     uintptr_t stack_addr;
     const uint8_t* stack;
@@ -807,7 +807,7 @@
 #if defined(__mips__)
   memcpy(&pr.pr_reg, &thread.mcontext.gregs, sizeof(user_regs_struct));
 #else
-  memcpy(&pr.pr_reg, &thread.regs, sizeof(user_regs_struct));
+  memcpy(&pr.pr_reg, &thread.regs, sizeof(user_pt_regs));
 #endif
 
   Nhdr nhdr;
